addon/components/step-manager.js

```diff
@@ -170,8 +187,34 @@ export default Component.extend({
     },
 
     /**
+     * Used internally to transition to a specific named step
+     *
+     * @method make-transition
+     * @param {string} to the name of the step to transition to
+     * @param {string} from the name of the step being transitioned
+     * @param {*} value the value to pass to the transition actions
+     * @private
+     */
+    'make-transition'(to, from, value) {
```

Comment:
Since this is only an internally-used method, can we make it a method on the component rather than an action? Just to make it clear that it's not invoked through the template.

Reply:
Yeah you're right, I also did think about that. I also thought that maybe we could expose this action in future to force transition the component. I don't know what do you think ? Maybe a better name could be used.

Reply:
Let's make it an internal method for now. I'd be hesitant to add it as a way to force a transition and ignore validation, if that's your idea. If the user has configured validations that they don't want to run, then they shouldn't have put the validations in there in the first place (or provide a way to skip validation other than invoking a different action)